x/dex/keeper/msg_server.go

```diff
@@ -152,6 +152,11 @@ func (k MsgServer) PlaceLimitOrder(
 		return &types.MsgPlaceLimitOrderResponse{}, err
 	}
 
+	ctx.EventManager().EmitEvents(types.GetEventsIncTotalOrders(&types.TradePairID{
```

Comment:
This should be removed. You are already incrementing in the correct place in keeper.limit_order_tranche.go